pkg/apis/extensions/v1beta1/types.go

```diff
@@ -213,6 +213,10 @@ type DeploymentSpec struct {
 	// Value of this key is hash of DeploymentSpec.PodTemplateSpec.
 	// No label is added if this is set to empty string.
 	UniqueLabelKey *string `json:"uniqueLabelKey,omitempty"`
+
+	// Paused indicates that the deployment will not be processed by the deployment
```

Comment:
Well, client-side scaling intentionally updates only the deployment to avoid any races (ie the only entrypooint for scaling the active RC is the deployment controller) so yes, paused blocks scaling behavior since the deployment won't be reconciled.